src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFliteTagMercury.java

```diff
@@ -0,0 +1,36 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.mobile.MobileTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.testng.annotations.Test;
+
+/**
+ * @ownership: AdEngineering Wikia
```

Comment:
I decided to do in this ticket AdEng (for consistency now) but created ADEN-2577 for change in all tests.